.github/renovate.json5

```diff
@@ -112,5 +112,18 @@
       ],
       datasourceTemplate: 'npm',
     },
+    {
+      customType: 'regex',
+      fileMatch: ['.md$'],
+
+      // Matches Buildkite plugins but preserves versions in the before diffs
+      // - docker-compose#v2.0.0: PASS
+      // - - docker-compose#v2.0.0: FAIL
+      // + - docker-compose#v2.0.0: PASS
+      matchStrings: [
+        '^[s+]*- (?<depName>[a-zA-Z0-9-]+?)#v(?<currentValue>.*?):',
```

Comment:
Should the `depName` pattern include `/` for e.g. `seek-oss/docker-ecr-cache`?

Reply:
Oh yes.